inc/saiswitch.h

```diff
@@ -1510,6 +1510,17 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_PFC_TC_DLD_INTERVAL_RANGE,
 
+    /**
+     * @brief PFC Deadlock Detection timer polling interval in milliseconds.
+     *
+     * PFC Deadlock Detection timer polling interval
+     *
+     * @type sai_map_list_t
```

Comment:
why the type is a map_list?

Reply:
should also define a range for the polling interval.

Reply:
please move after SAI_SWITCH_ATTR_PFC_TC_DLD_INTERVAL as SAI_SWITCH_ATTR_PFC_TC_DLD_INTERVAL and SAI_SWITCH_ATTR_PFC_TC_DLD_INTERVAL_RANGE are related.

Reply:
The type indeed should not be a map list. but for brcm,  the polling interval is not a range and is an enum.
typedef enum bcm_switch_pfc_deadlock_detection_interval_e {
    bcmSwitchPFCDeadlockDetectionInterval100MiliSecond = 0, /* 100 ms interval for Detection
                                           interval (default) */
    bcmSwitchPFCDeadlockDetectionInterval10MiliSecond = 1, /* 10ms interval for Detection interval */
    bcmSwitchPFCDeadlockDetectionIntervalCount = 2 /* Maximum PFC deadlock detecion type
                                           count */
} bcm_switch_pfc_deadlock_detection_interval_t;

Reply:
The define is modified as belows. Is it ok?

    SAI_SWITCH_ATTR_PFC_DLR_PACKET_ACTION,

    /**
     * @brief PFC Deadlock Detection timer polling interval range in milliseconds
     *
     * @type sai_u32_range_t
     * @default READ_ONLY
     */
    SAI_SWITCH_ATTR_PFC_TC_DLD_TIMER_INTERVAL_RANGE,

    /**
     * @brief PFC Deadlock Detection timer polling interval in milliseconds.
     *
     * PFC Deadlock Detection timer polling interval
     *
     * @type sai_uint32_t
     * @flags CREATE_AND_SET
     * @default empty
     */
    SAI_SWITCH_ATTR_PFC_TC_DLD_TIMER_INTERVAL,

    /**
     * @brief  PFC Deadlock Detection timer interval range
     *
     * @type sai_u32_range_t
     * @flags READ_ONLY
     */
    SAI_SWITCH_ATTR_PFC_TC_DLD_INTERVAL_RANGE,

Reply:
The map list is required to specify a value per priority just like - SAI_SWITCH_ATTR_PFC_TC_DLD_INTERVAL, SAI_SWITCH_ATTR_PFC_TC_DLR_INTERVAL.